packages/costa/src/lrucache.ts

```diff
@@ -0,0 +1,27 @@
+export default class LruCache<T> {
+  private values: Map<string, T> = new Map<string, T>();
```

Comment:
```suggestion
  private values = new Map<string, T>();
```
The type of values can derivate by the right value.